code/modules/clothing/suits/ego_gear/teth.dm

```diff
@@ -23,3 +23,15 @@
 	Sometimes, the wielder may see things we have long forgotten."
 	icon_state = "fragment"
 	armor = list(RED_DAMAGE = 0, WHITE_DAMAGE = -20, BLACK_DAMAGE = 50, PALE_DAMAGE = 0)
+
+/*Extraction officer coat.
+I don't want them gaming ego right off the bat beacuse this one actually looks pretty okay.
+It's not great though.
+*/
+/obj/item/clothing/suit/armor/ego_gear/extraction
```

Comment:
I'd suggest you move it outside of EGO gear, since it isn't using ANY of the EGO features, just make it a normal suit.

Reply:
Done